### PR TITLE
chore: include color helper

### PR DESCRIPTION
**Pull request by kaf-lamed-beyt**
#354 

Hi @argyleink, kindly take a look.

### CODE REVIEW

**Review by argyleink — CHANGES_REQUESTED** [review]
Looks great!

Can you update a style too, so it aligns better on desktop? Here's what needs changed.

```css
footer dl > div {
  place-self: start center;
}
```

**Comment by kaf-lamed-beyt** [comment]
> Can you update a style too, so it aligns better on desktop? Here's what needs changed.

All done.

Learned something new today, because this is the first time I'm seeing this particular CSS property &mdash; `place-self`

Thank you for the opportunity, @argyleink!

File: docsite/index.css [hunk 1]
```diff
@@ -169,6 +169,10 @@ footer {
   }
 }
 
+footer dl > div {
+  place-self: start center;
+}
+
```
Comment (argyleink) [inline docsite/index.css:175]: can you change it [here instead](https://github.com/argyleink/open-props/blob/main/docsite/index.css#L159) please =)

**Review by argyleink — CHANGES_REQUESTED** [review]

File: docsite/index.css (hunk 1 above)
Comment (kaf-lamed-beyt) [reply docsite/index.css:175]: I had a guess about this too. I'll change it now.

Comment (kaf-lamed-beyt) [reply docsite/index.css]: done

**Review by argyleink — APPROVED** [review]
ty! 💯

**Comment by kaf-lamed-beyt** [comment]
🫡

**Pull request merged**